online-shop/src/app/store/actions/order.actions.ts

```diff
@@ -0,0 +1,10 @@
+import {createAction, props} from "@ngrx/store";
+import {ProductQuantityModel} from "../../types/ProductQuantity.model";
+
+export const actualizeOrder = createAction('[ADD PRODUCT] Add Product To Order', props<{ productID: string }>());
+
+export const saveOrder = createAction('[Checkout] Checkout', props<{ products: ProductQuantityModel[] }>());
+
+export const saveOrderSuccess = createAction('[Checkout] Checkout Success');
+
+export const saveOrderFailure = createAction('[Checkout] Checkout Failure', props<{ error: any }>());
```

Comment:
You shouln't use 'any'. Think of what errors might occur, and define a type/ interface for the error. Maybe something like { error: OrderError } where interface OrderError { message: string }. Or whatever you want, it's up to you